### PR TITLE
Adjust trend analysis interpolation behavior

### PR DESCRIPTION
**Pull request by mdeceglie**
- [x] Code changes are covered by tests
- ~New functions added to `__init__.py`~
- ~API.rst is up to date, along with other sphinx docs pages~
- [x] Example notebooks are rerun and differences in results scrutinized
- ~[ ] Updated changelog~

### CODE REVIEW

**Comment by mdeceglie** [comment]
`TrendAnalysis` hasn't been part of a non-beta release, so I'm leaning toward skipping a separate change log entry for this, but I could be talked into including it.

File: rdtools/test/analysis_chains_test.py [hunk 1]
```diff
@@ -78,6 +78,48 @@ def sensor_analysis_exp_power(sensor_parameters):
     return rd_analysis
 
 
+def test_interpoloation(basic_parameters, degradation_trend):
```
Comment (kanderso-nrel) [inline rdtools/test/analysis_chains_test.py:81]:
```suggestion
def test_interpolation(basic_parameters, degradation_trend):
```

**Review by kanderso-nrel — COMMENTED** [review]
This seems like a good approach to me.  Right now the test only covers the interpolations in the constructor, so should we do the same index checks for the series involved in the `set_clearsky` method as well?

**Review by kanderso-nrel — APPROVED** [review]
LGTM

**Pull request merged**